src/main/java/com/personal/financeManager/documentProcessor/models/TransactionBean.java

```diff
@@ -0,0 +1,25 @@
+package com.personal.financeManager.documentProcessor.models;
+
+import java.util.Map;
+
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.NotEmpty;
+
+import lombok.Data;
+
+@Data
+public class TransactionBean {
+
+	 @NotBlank(message = "transactionID cannot be null or blank")
+	 private String transactionID;
```

Comment:
please check the notion, the **transactionID** is optional, so please remove **@NotBlank** validation.